website/static/js/profile.js

```diff
@@ -924,6 +929,11 @@ ListViewModel.prototype.unserialize = function(data) {
         self.editable(false);
     }
     self.contents(ko.utils.arrayMap(data.contents || [], function (each) {
+        for (var attr in each) {
```

Comment:
each in this case is an object, so I'm using a for-in loop to loop over the attributes of an object. Is there a better way to do that? I believe for-each loops only work with arrays